### PR TITLE
Add menu item "Show in explorer" in Resolve merge conflicts window

### LINKED ISSUE #9995: Add menu item "Show in explorer" in Resolve merge conflicts window
### Feature description

Sometimes need to just open folder where file
image

### Environment

- Git Extensions 3.5.4.12724
- Build 65f01f39982ccb121c4df7d4b00b506939553958
- Git 2.36.1.windows.1
- Microsoft Windows NT 10.0.19044.0
- .NET Framework 4.8.4510.0
- DPI 96dpi (no scaling)

### PR DESCRIPTION
**Pull request by rhumborl**
Fixes #9995 

## Proposed changes

- Add "Show in folder" to merge conflict, using existing SelectPathInFileExplorer method
- Also fix doc comment on ToNativePath, this was the wrong way round and confused me

### Before

image

### After

image

## Test methodology

- Not unit testable
- Tested clicking menu item on
  - Filename without spaces
  - Filename containing spaces
- Item is unclickable with multiple files selected

## Test environment(s)

- GIT 2.36.1.windows.1 (but really N/A)
- Windows 11 Pro 22000.675
  - re-using existing functionality so confident on other OSs

Only tested on English (UK)

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: GitCommands/PathUtil.cs [hunk 1]
```diff
@@ -19,14 +19,14 @@ public static class PathUtil
...
-        /// <summary>Replaces native path separator with posix path separator.</summary>
+        /// <summary>Replaces native path separator with posix path separator (/).</summary>
         [return: NotNullIfNotNull("path")]
         public static string? ToPosixPath(this string? path)
         {
             return path?.Replace(NativeDirectorySeparatorChar, PosixDirectorySeparatorChar);
         }
 
-        /// <summary>Replaces '\' with '/'.</summary>
+        /// <summary>Replaces '/' with native path separator.</summary>
```
Comment (RussKie) [inline GitCommands/PathUtil.cs:29]: I'm not so sure about "\". IIRC we convert all paths from `C:\path\path` to `C:/path/path`.

File: GitUI/CommandsDialogs/FormResolveConflicts.Designer.cs [hunk 2]
```diff
@@ -273,6 +275,13 @@ private void InitializeComponent()
             this.openWithToolStripMenuItem.Text = "Open With";
             this.openWithToolStripMenuItem.Click += new System.EventHandler(this.openWithToolStripMenuItem_Click);
             // 
+            // openFolderToolStripMenuItem
+            // 
+            this.openFolderToolStripMenuItem.Name = "openFolderToolStripMenuItem";
+            this.openFolderToolStripMenuItem.Size = new System.Drawing.Size(195, 22);
+            this.openFolderToolStripMenuItem.Text = "Open Containing Folder";
```
Comment (RussKie) [inline GitUI/CommandsDialogs/FormResolveConflicts.Designer.cs:282]: Please align the text with the existing wording, e.g.,
image

Adding an icon would be a good move too.

We also use the sentence casing.

**Review by RussKie — COMMENTED** [review]
👍 with the comments.

File: GitCommands/PathUtil.cs (hunk 1 above)
Comment (rhumborl) [reply GitCommands/PathUtil.cs:29]: ToPosixPath replaces \ with / as you say.
This is ToNativePath which does the reverse.

File: GitUI/CommandsDialogs/FormResolveConflicts.Designer.cs (hunk 2 above)
Comment (rhumborl) [reply GitUI/CommandsDialogs/FormResolveConflicts.Designer.cs]: Fixed - have updated the screenshot in the original description

File: GitCommands/PathUtil.cs (hunk 1 above)
Comment (RussKie) [reply GitCommands/PathUtil.cs:29]: Thanks for checking.

**Review by RussKie — APPROVED** [review]

**Comment by RussKie** [comment]
Thank you.

**Pull request merged**